bibliography.bib

```diff
@@ -8842,4 +8830,198 @@ @InProceedings {ZieMaaBagDey2008aaai
+    doi          = {10.1145/3067695.3082463},
+    note         = {},
+    annote       = {},
+}
+
+@Article {CulMou2016EC,
+    title       = {Evolving a Behavioral Repertoire for a Walking Robot},
+    author      = Cully_A # and # Mouret_JB,
+    journal     = EC,
+    series      = {},
+    volume      = {24},
+    number      = {1},
+    year        = {2016},
+    pages       = {59--88},
+    doi         = {10.1162/EVCO_a_00143},
+    note        = {},
+    annote      = {},
+}
+
+@Article {VasChaMou2018IEEETEVC,
+    title       = {Using Centroidal Voronoi Tessellations to Scale Up the Multidimensional Archive of Phenotypic Elites Algorithm},
+    author      = Vassiliades_V # and # Chatzilygeroudis_K # and # Mouret_JB,
+    journal     = IEEETEVC,
+    series      = {},
+    volume      = {22},
+    number      = {4},
+    year        = {2018},
+    pages       = {623--630},
+    doi         = {10.1109/TEVC.2017.2735550},
+    note        = {},
+    annote      = {},
+}
+
+@Misc {HasLigBir2023SWEVO-supp,
+    title        = {Automatic modular design of robots swarms based on repertoires of behaviors generated via novelty search: supplementary material},
+    author       = Hasselmann_K # and # Ligot_A # and # Birattari_M,
+    howpublished = {\url{https://iridia.ulb.ac.be/supp/IridiaSupp2022-002}},
+    year         = {2022},
+    note         = {},
+    annote       = {},
+}
+
+@Book {May2018book,
+    title        = {Gods and Robots: Myths, Machines, and Ancient Dreams of Technology},
+    author       = Mayor_A,
+    editor       = {},
+    publisher    = pub_PUP,
+    address      = Princeton_NJ_USA,
+    year         = {2018},
+    edition      = {},
+    doi          = {10.2307/j.ctvc779xn},
+    note         = {},
+    annote       = {},
+}
+
+@Article {XinXiaZhe-etal2022SCIROB,
+    title        = {Swarm of micro flying robots in the wild},
+    author       = Xin_Z # and # Xiangyong_W # and # Zhepei_W # and # Yuman_G # and # Haojia_L # and # Qianhao_W # and # Tiankai_Y # and # Haojian_L # and # Yanjun_C # and # Chao_X # and # Fei_G,
+    journal      = SCIROB,
+    series       = {},
+    volume       = {7},
+    number       = {66},
+    year         = {2022},
+    pages        = {eabm5954},
+    doi          = {10.1126/scirobotics.abm5954},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings {BreMon2010ppsn,
+    title        = {Environment-driven embodied evolution in a population of autonomous agents},
+    author       = Bredeche_N # and # Montanier_JM,
+    booktitle    = ppsn_2010,
+    editor       = Schaefer_R # and # Cotta_C # and # Kolodziej_J # and # Rudolph_G,
+    series       = srs_LNCS,
+    volume       = {6239},
+    number       = {},
+    year         = {2010},
+    pages        = {290--299},
+    publisher    = pub_SP,
+    address      = Berlin_DEU,
+    doi          = {10.1007/978-3-642-15871-1_30},
+    note         = {},
+    annote       = {},
+}
+
+@Article {MonCarBre2016FRAI,
```

Comment:
Format is not consistent with previous entries from FRAI: what here appears as "number" is "pages" in previous entries. I think this is the correct way to do it, since the pages appear explicitly in the pdf. I have added a general comment to reflect this